Content.Server/AlertLevel/AlertLevelPrototype.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Specialized;
+using Content.Shared.Sound;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.AlertLevel;
+
+[Prototype("alertLevels")]
+public sealed class AlertLevelPrototype : IPrototype
+{
+    [IdDataField] public string ID { get; } = default!;
+
+    /// <summary>
+    /// Dictionary of alert levels. Keyed by string - the string key is the most important
+    /// part here. Visualizers will use this in order to dictate what alert level to show on
+    /// client side sprites, and localization uses each key to dictate the alert level name.
+    /// </summary>
+    [DataField("levels")] public Dictionary<string, AlertLevelDetail> Levels = new();
+
+    /// <summary>
+    /// Default level that the station is on upon initialization.
+    /// If this isn't in the dictionary, this will default to whatever .First() gives.
+    /// </summary>
+    [DataField("defaultLevel")] public string DefaultLevel { get; }= default!;
+}
+
+/// <summary>
+/// Alert level detail. Does not contain an ID, that is handled by
+/// the Levels field in AlertLevelPrototype.
+/// </summary>
+[DataDefinition]
+public sealed class AlertLevelDetail
+{
+    /// <summary>
+    /// What is announced upon this alert level change. Can be a localized string.
+    /// </summary>
+    [DataField("announcement")] public string Announcement { get; } = default!;
```

Comment:
nullable string?

Reply:
nah, it should still be at the very least an empty string if there's no real announcement - maybe `default!` isn't the best thing here and string.Empty should be the better replacement?

Reply:
yeah I'd just go with a string.Empty and make it a required field then.

Reply:
it's theoretically fine to have an empty announcement, but it's just lazy